models/buku.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Buku extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+  Buku.init({
+    title: DataTypes.STRING,
+    status: DataTypes.STRING,
+    description: DataTypes.TEXT
+  }, {
+    sequelize,
+    modelName: 'Buku',
+  });
```

Comment:
belum ada hooks and validation nya disini, validation title gabisa lebih dari 255, hooks untuk status selalu jadi created dan semacamnya